### PR TITLE
[#160648724, #160648720] Fix order of displayed cards

### PR DESCRIPTION
**Pull request by fgiobergia**
There previously was no established order among the cards provided by pagoPA. The relevant stories for this PR introduce an order (most recently used first), which has been introduced accordingly in the app.

### CODE REVIEW

File: ts/sagas/wallet.ts [hunk 1]
```diff
@@ -251,42 +251,42 @@ function* addCreditCard(
...
-      idWallet,
-      tipo: "web",
-      cvv: wallet.creditCard.securityCode
-        ? wallet.creditCard.securityCode
-        : undefined
-    }
-  };
   try {
     yield put(walletManagementSetLoadingState());
+    const wallet: NullableWallet = {
```
Comment (fgiobergia) [inline ts/sagas/wallet.ts]: Not exactly relevant to this story, but part of the asynchronous logic in the "add card" function was performed before showing the spinner

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/sagas/wallet.ts [hunk 2]
```diff
@@ -251,42 +251,41 @@ function* addCreditCard(
...
+    const responseBoardCC:
+      | BasicResponseTypeWith401<WalletResponse>
+      | undefined = yield call(
+      fetchWithTokenRefresh,
+      (token: string) => pagoPaClient.boardCreditCard(token, wallet),
+      pagoPaClient
+    );
+
+    /**
+     * Failed request. show an error (TODO) and return
```
Comment (cloudify) [inline ts/sagas/wallet.ts]: this this TODO part of this story? if not, reference the story

**Comment by fgiobergia** [comment]
Not WIP as this story is not concerned with error handling. Added the relevant story to the comment

**Pull request merged**